Provide more context in error messages

If the `redpanda.adminAPI.urls` field is empty, Redpanda Console throws this error:

```
panic: runtime error: index out of range [0] with length 0

goroutine 1 [running]:
github.com/redpanda-data/console/backend/pkg/redpanda.NewService({{0x1, {0x0, 0x0, 0x0}, {0x0, 0x0}, {0x0, 0x0}, {0x0, {0x0, ...}, ...}}}, ...)
	/go/pkg/mod/github.com/redpanda-data/console/backend@v0.0.0-20220810143308-dc4e1eaccd53/pkg/redpanda/service.go:44 +0x750
github.com/redpanda-data/console/backend/pkg/api.New(0xc000405600, {0xc0005bf2a0, 0x1, 0x8?})
	/go/pkg/mod/github.com/redpanda-data/console/backend@v0.0.0-20220810143308-dc4e1eaccd53/pkg/api/api.go:60 +0x425
main.main()
	/app/cmd/main.go:67 +0xc85
```

It would be nice to try to catch this and provide the user with more context.

--- a/backend/pkg/config/redpanda_admin_api.go
+++ b/backend/pkg/config/redpanda_admin_api.go
@@ -40,6 +40,10 @@ func (c *RedpandaAdminAPI) Validate() error {
 		return nil
 	}
 
+	if len(c.URLs) == 0 {
+		return fmt.Errorf("you must specify at least one URL")
+	}
+
 	for _, u := range c.URLs {
 		urlParsed, err := url.Parse(u)
 		if err != nil {

